### PR TITLE
Fixup

### PR DESCRIPTION
**Pull request by alicebartlett**
This PR:

- Rearranges the demos a bit, I've tried to remove stuff we don't need to demo, and add in some better descriptions of what the demos are for
- Rearranges the readme. The readme had a lot of stuff in, but this made it very difficult to keep up to date. I hope this will be easier to maintain.

@onishiweb

### CODE REVIEW

File: origami.json [hunk 1]
```diff
@@ -1,73 +1,76 @@
...
+    "dependencies": [
+      "o-fonts@^1.4.0"
+    ]
+  },
+  "demos": [
+    {
+      "name": "standard",
+      "title": "Masterbrand standard theme",
+      "template": "/demos/src/standard.mustache",
+      "description": "Standard theme for masterbrand buttons. This is the defaul theme."
```
Comment (onishiweb) [inline origami.json]: default (missing the `t`)

File: origami.json [hunk 2]
```diff
@@ -1,73 +1,76 @@
...
     },
-    "demosDefaults": {
-      "sass": "demos/src/demo.scss",
-      "js": "demos/src/demo.js",
-      "dependencies": [
-        "o-fonts@^1.4.0"
-      ]
+    {
+      "name": "standout",
+      "title": "Masterbrand stanout theme",
```
Comment (onishiweb) [inline origami.json]: standout (missing a `d`)

File: designguidelines.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
+#### Themes
+
+* __default__: teal outline
+* __standout__: solid teal
+* __uncolored__: monochrome
+* __inverse__: for use on dark backgrounds
+* __b2c__: A theme for b2c products eg http://help.ft.com
```
Comment (onishiweb) [inline designguidelines.md]: Could this be marked up as a link?

File: designguidelines.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+and the following sizes ([documentation](#size-modifiers)):
```
Comment (onishiweb) [inline designguidelines.md]: Would this be better as `and the following sizes:` with the link removed, as it won't go back to the repo docs

File: README.md [hunk 5]
```diff
@@ -233,20 +219,32 @@ Or, using Sass:
 }
 
 .my-button-class--icon-star {
-	// icon here can be *any* icon tag (eg arrow-left) in o-ft-icons
+	// icon here can be *any* icon tag (eg arrow-left) in o-icons
```
Comment (onishiweb) [inline README.md]: Would it be better to reference `fticons` here?

**Comment by alicebartlett** [comment]
Wow i'm badf ar tuyping huh.
Thanks for the feedback, I've made amendments.

File: README.md (hunk 5 above)
Comment (onishiweb) [inline README.md]: @alicebartlett last thing

**Pull request merged**